src/utils/utils.ts

```diff
@@ -0,0 +1,105 @@
+import { cards } from "../cards";
+import { Card } from "../types/Card";
+import findIndex from "lodash.findindex";
+import isEqual from "lodash.isequal";
+
+const randNum = () => Math.ceil(15 * Math.random());
+
+// const randCard = () => {
+//   const randIdx = randNum();
+//   return cards[randIdx];
+// };
+
+export const initDeck = () => {
+  let newDeck: number[] = [];
+
+  while (newDeck.length < 12) {
+    const newNum = randNum();
+
+    if (!newDeck.includes(newNum)) {
+      newDeck.push(newNum);
+    }
+  }
+  return newDeck.map((number) => {
+    const newCard = cards[number - 1];
+    return newCard;
+  });
+};
+
+const cardInDeck = (deck: Card[], cardId: number) => {
```

Comment:
Maybe we can rename this function to `isCardInDeck` so the intent is more explicit, what do you think?